functions.js

```diff
@@ -0,0 +1,50 @@
+const ourTest = {
+    capitalize: (str) => {
+        return `${str[0].toUpperCase()}${str.substr(1, str.length-1)}`;
+    },
+    reverse: (str) => {
+        return str.split("").reverse().join("");
+    },
+    calculator: {
+        add: (x, y) => x + y,
+        subtract: (x, y) => x - y,
+        divide: (x, y) => x / y,
+        multiply: (x, y) => x * y,
+    },
```

Comment:
opps, you test should handle positive and negative scenario.
if I introduce a bug on the calculation methods test still passes
example: `line 9` I changed `x+y` to `x*y`, to my surprise test still passes.

Reply:
Hi Bello,

Thanks for your feedback, for this particular test I think we were just to test for the presence of the methods inside the object and not necessarily the calculations

Reply:
Hi @MarvellousUbani, 

Yes, you might think like that but this project basically teaching us how to do testing in jest right and I think one of the core reasons why we write tests is to mitigate the time spend on debugging, so we can easily fix the methods that failed on our test result.

According to Odin instructions, you are to test all functions;
![image](https://user-images.githubusercontent.com/31897434/85220621-00a4d680-b3a5-11ea-8577-67227a404380.png)
https://www.theodinproject.com/courses/javascript/lessons/testing-practice

I hope this helps!
Happy Coding 👍